### DISCUSSION TITLE
config help - do not group all changes for a dependency in one PR

### DISCUSSION THREAD

**Discussion by mcooknu** [renovatebot/renovate#28889] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

self-hosting k8s, v37.347.0

### Please tell us more about your question or problem

Renovatebot is creating PRs for Terraform projects but grouping by dependency across multiple projects.

We would like to not group, as applying a single PR for all three dev, stg, and prd environments not permitted.  
Instead, what we want is one PR per instance of the dependency.

Is this possible and what would we have to write in the renovate.config or renovate.json packageRules to make this happen?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
It's possible, please see the answer in https://github.com/renovatebot/renovate/discussions/28505

**Reply by mcooknu** [reply]
Thanks for the hint.

I successfully got what I wanted using these settings in a custom manager...

"packageRules": [
        {
...
            "labels": ["My Label"],
            "additionalBranchPrefix": "{{baseDir}}-",
            "commitMessageSuffix": "({{baseDir}})",
>>>
